docs/docs/data/expressions-related.json

```diff
@@ -0,0 +1,220 @@
+{
```

Comment:
Is this needed for the style-spec docs?
Can we somehow incorporate this in the v8.json file?